src/main.js

```diff
@@ -336,19 +336,36 @@ class MiniGraphCard extends LitElement {
       </div>` : '';
   }
 
+  generateLegend(index) {
+    let legend = this.computeName(index);
+    const state = this.getEntityState(index);
+
+    const show_legend_state = this.config.entities[index].show_legend_state ?? false;
+
+    if (show_legend_state) {
+      legend += ` (${this.computeState(state)} ${this.computeUom(index)})`;
+    }
+
+    return legend;
+  }
+
   renderLegend() {
     if (this.visibleLegends.length <= 1 || !this.config.show.legend) return;
+
     return html`
       <div class="graph__legend">
-        ${this.visibleLegends.map(entity => html`
-          <div class="graph__legend__item"
-            @click=${e => this.handlePopup(e, this.entity[entity.index])}
-            @mouseenter=${() => this.setTooltip(entity.index, -1, this.getEntityState(entity.index), 'Current')}
-            @mouseleave=${() => (this.tooltip = {})}>
-            ${this.renderIndicator(this.getEntityState(entity.index), entity.index)}
-            <span class="ellipsis">${this.computeName(entity.index)}</span>
-          </div>
-        `)}
+        ${this.visibleLegends.map((entity) => {
+    const legend = this.generateLegend(entity.index);
+    return html`
```

Comment:
Please indent properly.

Reply:
I tried but eslint throw an error if it is not like so.
To fix this, I've also tried to  ignore html templates, but it made the rest of the project wrong by eslint standard

Reply:
That made me curious and I went down that painful road myself, too... I couldn't come up with something other than ignoring the rule for this single template using: 

```js
/* eslint-disable indent */
...
/* eslint-enable indent */
```

I would be fine to cheat eslint this time. What do you think?

Reply:
fixed